frontend/src/lib/taxonomy.tsx

```diff
@@ -230,6 +235,31 @@ export const KEY_MAPPING: KeyMappingInterface = {
             description: 'The initial type of device that was used (first-touch).',
             examples: ['Mobile', 'Tablet', 'Desktop'],
         },
+        $screen_density: {
+            label: 'Screen density',
+            description: 'WHAT IS THE DEFINITION?',
```

Comment:
```suggestion
            description: 'The logical density of the display. This is a scaling factor for the Density Independent Pixel unit, where one DIP is one pixel on an approximately 160 dpi screen (for example a 240x320, 1.5"x2" screen), providing the baseline of the system's display. Thus on a 160dpi screen this density value will be 1; on a 120 dpi screen it would be .75; etc.',
```

Just got it from official docs.